book/src/SUMMARY.md

```diff
@@ -5,9 +5,18 @@
 - [Terminology](terminology.md)
 
 - [Getting Started](getting-started.md)
-  - [Testnet Participation](testnet-participation.md)
-  - [Testnet Replicator](testnet-replicator.md)
-  - [Example: Web Wallet](webwallet.md)
+  - [Running a Validator](running-validator.md)
```

Comment:
This is too detailed for the "Getting Started" section of the book.  This part of the book aims to be the minimum steps to get to that "Example Client" chapter.  Can you move it further down? Maybe after "Anatomy of a Transaction"?

Reply:
I'll do that, but it's unfortunate that it ends up below the fold in the navigation menu. (I'd actually like to see some design/usability improvements in the book, including collapsing menu items, which would resolve this particular issue.)